### ISSUE TITLE
[IntelliJ][Reader Account]There is a spelling mistake in the warning when submit with adls cluster.

### ISSUE THREAD

**Issue by v-xiaocw** [microsoft/azure-tools-for-java#3096]
Build: azure-toolkit-for-intellij-2019.1.develop.1112.05-05-2019
Repro Steps:
1.Sign in with a reader account that does not have adls cluster permission
2.Linked an adls cluster ,eg. zhouytestgen1
3.Create a run config with adls cluster and run the config
Result: There is a spelling mistake in the warning should change the "use" to "user"
![image](https://user-images.githubusercontent.com/43333874/57209383-5514a900-6f8d-11e9-9575-b311130d2a1e.png)

**Comment by t-rufang** [comment]
Fixed in #3098

**Comment by jingyanjingyan** [comment]
Fix in dev 1137, the error message display as below:

```
Package and deploy the job to Spark cluster
ERROR:  [60797797-a491-4551-9e7f-32d07fbd6169] Access Denied : /clusters2/SparkSubmission/2019/05/30/88ade881-de12-4dab-8b3c-33f61bf7a796/default_artifact.jar[2019-05-29T20:36:42.4598735-07:00]
Please make sure user has RWX permissions for the storage account
```
